app/views/tracks/concepts/show.html.haml

```diff
@@ -17,22 +17,18 @@
     %div
       .about
         %h2 About #{@concept.name}
-        %p C# structs are closely related to classes. They have state and behavior. They have constructors that take arguments, instances can be assigned, tested for equality and stored in collections.
+        %p
+          = @concept.about
 
       .links
         %h3 Learn More
         %ul
-          %li
-            = link_to "#" do
-              = bg_img("https://c.s-microsoft.com/favicon.ico?v2", "Favicon of linked website")
-              Structs in C#
-              = icon(:external_link, "(external link)")
-
-          %li
-            = link_to "#" do
-              = bg_img("https://c.s-microsoft.com/favicon.ico?v2", "Favicon of linked website")
-              Using Methods in C#
-              = icon(:external_link, "(external link)")
+          - @concept.links.each do |link|
+            %li
+              = link_to link['url'] do
+                = bg_img("https://c.s-microsoft.com/favicon.ico?v2", "Favicon of linked website")
+                = link['description']
+                = icon(:external_link, "(external link)")
```

Comment:
It would be nice if link wasn't a hash, but instead was an object. You could achieve this by mapping each links to be an openstruct.

Reply:
Done